raiden-ts/src/transport/epics/webrtc.ts

```diff
@@ -51,6 +53,31 @@ import { waitMemberAndSend$, parseMessage } from './helpers';
 
 type CallInfo = { callId: string; peerId: string; peerAddress: Address };
 
+const RtcOffer = t.readonly(
+  t.intersection([
+    t.type({ type: t.literal('offer'), sdp: t.string }),
+    t.partial({ call_id: t.string }),
```

Comment:
Additional question: why do we call this `call_id`. In my understanding webRTC is a generic data channel, so `id` would be more fitting...

Reply:
`call_id` is optional in the new algorithm, and used only for debugging, not required (but still recommended) to be sent via signaling since each end calculates it deterministically.
We call it `call_id` to minimize the implementation difference from the previous algorithm. Yes, only the generic data channel part of the webRTC is used here, but on webRTC jargon, this is still considered a call (a data call).